example/gateway/apq-client/index.js

```diff
@@ -0,0 +1,45 @@
+const { ApolloClient, InMemoryCache, gql, HttpLink } = require('@apollo/client');
```

Comment:
This did not work for me, what i did to fix that was:

1. `const fetch = require('cross-fetch');`
2. `const { ApolloClient, InMemoryCache, gql, HttpLink } = require('@apollo/client/core');`
3. `new HttpLink({ uri: 'http://localhost:4000',fetch}),`

Reply:
I see what you mean, I was running a client like this: `node --experimental-fetch index.js`.
I'll commit these fixes so that running the client is more straightforward.